plugin/insert-char.vim

```diff
@@ -3,7 +3,12 @@ function! <SID>InsertChar(char, count)
 endfunction
 
 nnoremap <silent> <Plug>InsertChar :<C-U>exec "normal i".<SID>InsertChar(nr2char(getchar()), v:count1)<CR>
+nnoremap <silent> <Plug>AppendChar :<C-U>exec "normal a".<SID>InsertChar(nr2char(getchar()), v:count1)<CR>
 
 if !exists('g:insert_char_no_default_mapping') || (g:insert_char_no_default_mapping == 0)
-  nmap <Space> <Plug>InsertChar
```

Comment:
Let's leave this as `<Space>`. It's not wise to change existing defaults without a good reason.